### ISSUE TITLE
latest tag for docker image is missing from Docker Hub

### ISSUE THREAD

**Issue by bwjohnson-ss** [wework/speccy#257]
```
% docker pull wework/speccy
Using default tag: latest
Error response from daemon: manifest for wework/speccy:latest not found
```

**Comment by bwjohnson-ss** [comment]
@philsturgeon  Any ideas what's wrong with this?

**Comment by philsturgeon** [comment]
I had removed it mistakenly, as part of #254. You can just use `:0` or `:0.8` in the mean time. 

The fix was merged in #255 and no that #256 is merged you should see `latest` has returned! 

https://hub.docker.com/r/wework/speccy/tags
